src/geovista/examples/scalar_data/earthquakes.py

```diff
@@ -39,6 +39,11 @@
 See `reference <https://www.mtu.edu/geo/community/seismology/learn/earthquake-measure/magnitude/>`_
 for further details.
 
+.. attention::
+
+   Optional package dependencies `pandas <https://pandas.pydata.org/docs/>`_ and
+   `fastparquet <https://fastparquet.readthedocs.io/en/stable>`_ are required.
```

Comment:
The `sphinx` `object.inv` for `fastparquet` does not contain a Python domain module level reference i.e., `py:mod` ... so for consistency of rendering in the docs, opting for URLs rather than `intersphinx`.